<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modif</class>
 <widget class="QDialog" name="modif">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #0d8ece;
font-size:50;
font:bold large;</string>
   </property>
   <property name="text">
    <string>Identifiant de l'animation </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>40</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tab_ab_rech_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>541</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Rechercher_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>40</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rechercher</string>
   </property>
  </widget>
  <widget class="QPushButton" name="trie_asc">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>180</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>trie asc</string>
   </property>
  </widget>
  <widget class="QTableView" name="tabanime">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>571</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="trie_desc">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>180</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>trie desc</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_duree_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_idanim_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #0d8ece;
font-size:50;
font:bold large;</string>
   </property>
   <property name="text">
    <string>Modifier Animation : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_cin_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_lieu_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dateanim_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_modifier">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_typeanim_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>301</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
